dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -5,49 +5,48 @@
 #NLS_MESSAGEFORMAT_NONE
 
 authFilter=Authentication Filter
-authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers to determine whether or not the HTTP request is selected for the authentication.
+authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication.
 
 webApp=Web Application
-webApp.desc=Specifies the web application name.
+webApp.desc=Specifies the web application name. For example, <webApp id="sample" name="app1|app2" matchType="notContain"/>
 
 requestUrl=Request URL
-requestUrl.desc=Specifies the request URL.
+requestUrl.desc=Specifies the HTTP request URL. For example, <requestUrl id="sample" urlPattern="/account/checking|/account/saving" matchType="contains" />
 
 urlPattern=URL pattern
-urlPattern.desc=Specifies the URL pattern.
-
-remoteAddress=Remote Host
-remoteAddress.desc=Specifies the remote host.
+urlPattern.desc=Specifies the URL pattern. The "*", when used as a wildcard, it is not supported.
```

Comment:
I'm not really sure what "it" is referring to in this one. 

Should it be: 
"Specifies the URL pattern. When the `*` character is used as a wildcard, the URL pattern is not supported."

Or maybe:
"Specifies the URL pattern. The `*` character is not supported to be used as a wildcard."

Reply:
"Specifies the URL pattern. The * character is not supported to be used as a wildcard."